pkg/config/config.go

```diff
@@ -406,5 +409,43 @@ func ParseConfig(cfg Getter) (*Config, error) {
 		}
 	}
 
+	sendDeletedClustersLogsToDefaultClient := cfg.Get("SendDeletedClustersLogsToDefaultClient")
+	if sendDeletedClustersLogsToDefaultClient != "" {
+		res.SendDeletedClustersLogsToDefaultClient, err = strconv.ParseBool(sendDeletedClustersLogsToDefaultClient)
+		if err != nil {
+			return nil, fmt.Errorf("invalid string SendDeletedClustersLogsToDefaultClient: %v", err)
+		}
+	}
+
+	deletedClientTimeExpiration := cfg.Get("DeletedClientTimeExpiration")
+	if deletedClientTimeExpiration != "" {
+		deletedClientTimeExpirationValue, err := strconv.Atoi(deletedClientTimeExpiration)
```

Comment:
1) You can user `func ParseDuration(str string) (Duration, error)` and to avoid this using integers.
2) It is not good idea silently to override values < 60s
3) Can you please, update the example configurations in docs and to set the new configs there